### PR TITLE
Elements with no role in aria-allowed-attribute

### PR DESCRIPTION
**Pull request by annethyme**
Expectation updated to handle elements that does not have a semantic role. 

Closes issue: https://github.com/auto-wcag/auto-wcag/issues/295

# How to Review And Approve
- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.

### CODE REVIEW

File: _rules/SC4-1-1+SC4-1-2-aria-allowed-attribute.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Any attribute that is specified on an HTML or SVG element where the attribute na
 
 ### Expectation
 
-The target attribute is either listed under [WAI-ARIA 1.1 Supported States and Properties](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties) for the [semantic role](#semantic-role) of the element that the attribute is specified on, or it is found on the list of [WAI-ARIA 1.1 Global States and Properties](https://www.w3.org/TR/wai-aria-1.1/#global_states).
+The target attribute is either found on the list of [WAI-ARIA 1.1 Global States and Properties](https://www.w3.org/TR/wai-aria-1.1/#global_states), or is listed under [WAI-ARIA 1.1 Supported States and Properties](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties) for the [semantic role](#semantic-role) of the element that the attribute is specified on, if the element has a semantic role.
```
Comment (WilcoFiers) [inline _rules/SC4-1-1+SC4-1-2-aria-allowed-attribute.md]: Can we break this up into two sentences?

Comment (WilcoFiers) [inline _rules/SC4-1-1+SC4-1-2-aria-allowed-attribute.md]: Can we break this up into two sentences?

Comment (WilcoFiers) [inline _rules/SC4-1-1+SC4-1-2-aria-allowed-attribute.md]: Can we break this up into two sentences?

Comment (WilcoFiers) [inline _rules/SC4-1-1+SC4-1-2-aria-allowed-attribute.md]: Can we break this up into two sentences?

Comment (WilcoFiers) [inline _rules/SC4-1-1+SC4-1-2-aria-allowed-attribute.md]: Can we break this up into two sentences?

Comment (WilcoFiers) [inline _rules/SC4-1-1+SC4-1-2-aria-allowed-attribute.md]: Can we break this up into two sentences?

Comment (WilcoFiers) [inline _rules/SC4-1-1+SC4-1-2-aria-allowed-attribute.md]: Can we break this up into two sentences?

**Comment by annethyme** [comment]
This PR conflicts with https://github.com/auto-wcag/auto-wcag/pull/311, so I will close this PR and update #311 to also include the change from this PR.

**Pull request closed without merging**